### DISCUSSION TITLE
When will it be updated to. NET 9

### DISCUSSION THREAD

**Discussion by nczsl** [dotnet/interactive#3756] [General]
After installing the new version and deleting the old version, it no longer works

**Comment by konst-sh** [comment]
I have net8 class lib project in which I included System.Text.Json 9.0.0 package, and after that I'm no longer able to reference that project dll in notebook as it always argues on assembly versions conflict

**Comment by jonsequitur** [comment]
The update (#3760) is in our November sprint plan.
